packages/select/src/index.mts

```diff
@@ -25,9 +25,14 @@ type Choice<Value> = {
   type?: never;
 };
 
+type SelectConfig<Value> = PromptConfig<{
+  choices: ReadonlyArray<Choice<Value> | Separator>;
+  pageSize?: number;
+}>;
+
 type Item<Value> = Separator | Choice<Value>;
 
-function isSelectableChoice<Value>(item: Item<Value>): item is Choice<Value> {
+function isSelectable<Value>(item: Item<Value>): item is Choice<Value> {
```

Comment:
I thought we used `isSelectableChoice` in #1299 so I renamed it specifically 😅

Reply:
Haha I figured, didn't want to bother and get this in today 😂